projects/reacher.email/check-if-email-exists-cli/package.yml

```diff
@@ -0,0 +1,29 @@
+distributable:
+  url: https://github.com/reacherhq/check-if-email-exists/archive/refs/tags/v{{ version }}.tar.gz
+  strip-components: 1
+
+provides:
+  - bin/check_if_email_exists
+
+versions:
+  github: reacherhq/check-if-email-exists
+  strip: /v/
+
+dependencies:
+  openssl.org: ^1.1
+
+build:
+  working-directory: cli
+  dependencies:
+    rust-lang.org: '>=1.65'
+    rust-lang.org/cargo: '*'
+    perl.org: '*'
+    tea.xyz/gx/make: '*'
+  script:
+    cargo install --path . --root {{prefix}}
+
+test:
+  script:
+    - which check_if_email_exists
+    # The CLI version is misaligned with the repo releases/tags
+    # - test "$(check_if_email_exists --version)" = "check-if-email-exists-cli 0.8.32"
```

Comment:
There's probably a way to fix this. I'd grep the sources for that version, then probably use sed to alter it during build.